x/pool-incentives/keeper/grpc_query.go

```diff
@@ -133,6 +133,10 @@ func (q Querier) IncentivizedPools(ctx context.Context, _ *types.QueryIncentiviz
 
 	// Loop over the distribution records and fill in the incentivized pools struct.
 	for _, record := range distrInfo.Records {
+		// Skip community pool gauge
+		if record.GaugeId == 0 {
```

Comment:
Asking this since I don't have the context behind it, does gauge id 0 **always** indicate that it is community pool? Can it also not mean that it's just a record without any gauge?

Reply:
Its a good question, yes in the context of distribution info, if a record is added with a gauge ID of zero, it is explicitly indicating to push incentives to the community pool.